apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -215,6 +221,19 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
           />
           <FormHelperText>{t('set-config.admin-password-help')}</FormHelperText>
         </FormControl>
+        <FormControl>
+          <FormLabel>{t('set-config.confirm-password')}</FormLabel>
+          <Input
+            type='password'
+            value={confirmPassword}
+            onChange={(ev) => setConfirmPassword(ev.currentTarget.value)}
+          />
+          <FormHelperText>
```

Comment:
We should use `FormErrorMessage`: https://chakra-ui.com/docs/components/form-control#error-message

Reply:
done